zuul.d/project-templates.yaml

```diff
@@ -1149,6 +1149,131 @@
               - ^lib/ansible/plugins/terminal/vyos.py
               - ^test/integration/targets/vyos_.*
               - ^test/integration/targets/prepare_vyos_tests/.*
+        - ansible-test-cloud-integration-vcsa-python36:
+            branches:
+              - devel
+            files:
```

Comment:
can we simplify this?

Maybe something like

^lib/ansible/modules/cloud/vmware/*
^lib/ansible/module_utils/vmware.*
 ^test/integration/targets/vcenter.*

to start

Reply:
Just a subset of the vmware modules can be tested with just one vcenter. So I'm not sure it makes sense to trigger the test if we change a module that is not covered. It sounds like just a waste of resources.

Reply:
I wouldn't say a waste of resources, once we move to collections, we'll be running this job all the time.  But yes, any changes to vmware, I'd say should trigger our vcsa job for now. There is a trade off, for me history has taught me, people tend to get confused if their jobs doesn't run, if the matcher gets out of sync with new files, etc. For that reason along, tend to make the matcher as large as possible.

But even here, ^lib/ansible/module_utils/vmware.* is duplicated a bunch of times, this isn't need. And possible a copypaste issue?

Reply:
Ok I will trigger the job for all the vmware changes. And indeed, the dup ^lib/ansible/module_utils/vmware.* comes from a bug in my script. Let's fix that.